lib/ruby_smb/dcerpc/wkssvc.rb

```diff
@@ -23,17 +24,99 @@ module Wkssvc
       WKSTA_INFO_102 = 0x00000066
       #TODO: WKSTA_INFO_502 = 0x000001F6
 
+      # User Enum Information Level
+      WKSTA_USER_INFO_0 = 0x00000000
+      WKSTA_USER_INFO_1 = 0x00000001
+
+      # [2.2.2.1 WKSSVC_IDENTIFY_HANDLE](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-wkst/9ef94a11-0e5c-49d7-9ac7-68d6f03565de)
+      class WkssvcIdentifyHandle < Ndr::NdrWideStringPtr; end
+
+      # [2.2.5.9 WKSTA_USER_INFO_0](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-wkst/b7c53c6f-8b92-4e5d-9a2e-6462cb4ef1ac)
+      class UserInfo0 < Ndr::NdrStruct
+        default_parameter byte_align: 4
+        endian :little
+
+        ndr_wide_stringz_ptr :wkui0_username
+      end
+
+      class WkstaUserInfo0 < Ndr::NdrConfArray
+        default_parameter type: :user_info0
+      end
+
+      class PwkstaUserInfo0 < WkstaUserInfo0
+        extend Ndr::PointerClassPlugin
+      end
```

Comment:
I think typically we'd combine these definitions. As it is right now `UserInfo0` should really be `WkstaUserInfo`.

```suggestion
      # [2.2.5.9 WKSTA_USER_INFO_0](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-wkst/b7c53c6f-8b92-4e5d-9a2e-6462cb4ef1ac)
      class WkstaUserInfo0 < Ndr::NdrStruct
        default_parameter byte_align: 4
        endian :little

        ndr_wide_stringz_ptr :wkui0_username
      end

      class WkstaUserInfo0ArrayPtr < Ndr::NdrConfArray
        default_parameter type: :wksta_user_info0
        extend Ndr::PointerClassPlugin
      end
```

I'm basing this on the precedence @cdelafuente-r7 set in `lib/ruby_smb/dcerpc/drsr.rb`.